### ISSUE TITLE
Confusion with documentation folder names

### ISSUE THREAD

**Issue by cmrd-senya** [diaspora/diaspora_federation#66]
Currently `diaspora_federation` has two documentation sets in it: API reference documentation and protocol documentation. API reference documentation is generated with `yardoc` and placed in `/doc` folder. Protocol documentation is rendered via `jekyll` and is placed under `/docs` folder. The folder names `doc` and `docs` make some confusion. I always forget which one is which and have to `ls` the folder to get it. Maybe we should change `doc` to `api_doc` or `api_reference` and `docs` to `protocol_doc` or `protocol_documentation`?

**Comment by SuperTux88** [comment]
The `/docs` folder is fix, because GitHub pages can only use this folder, see [here](https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/).

But feel free to change the output folder for yardoc. This documentation is published [here](http://www.rubydoc.info/gems/diaspora_federation/0.1.9) and I hope they don't depend on our `.yardopts` file, so I think we can change it there.

**Comment by cmrd-senya** [comment]
Maybe the following can work: folder `/docs` as a root for the documenation, and `/docs/protocol` and `/docs/api` as subfolders?

**Comment by SuperTux88** [comment]
No. GitHub Pages is `/docs` folder or `/` folder ... nothing else.

![spectacle b19107](https://cloud.githubusercontent.com/assets/458548/26532480/10879d0a-4402-11e7-9f2c-2542fbcad905.png)

**Comment by cmrd-senya** [comment]
If it was possible to tell jekyll to build site from a subfolder, then we could move it downwards to `/docs/protocol`. But I didn't find the way to do that. I'll check if it is possible to move the api reference to a separate folder.

**Comment by SuperTux88** [comment]
> I'll check if it is possible to move the api reference to a separate folder.

You can set `--output-dir` in `.yardopts`. I hope www.rubydoc.info doesn't have a problem with that, because I have no control over that.
